utility/src/main/java/org/oppia/android/util/logging/performancemetrics/PerformanceMetricsAssessorImpl.kt

```diff
@@ -70,4 +78,21 @@ class PerformanceMetricsAssessorImpl @Inject constructor(
       else -> HIGH_MEMORY_TIER
     }
   }
+
+  override fun getCpuUsage(): Double {
+    val clockSpeedHz = Os.sysconf(OsConstants._SC_CLK_TCK)
+    val uptimeSec = oppiaClock.getElapsedRealTime() / 1000.0
+    val cpuTimeSec = Process.getElapsedCpuTime() / 1000.0
+    val pid = Process.myPid()
+    val processTimeSec = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+      val startTimeSec = Process.getStartElapsedRealtime() / 1000.0
+      uptimeSec - startTimeSec
+    } else {
```

Comment:
Alternatively, we could just track the elapsed realtime since onCreate() and compute a slightly-off version of getStartElapsedRealtime. I think that might cause the CPU math to be slightly off, but it'd be isolated to pre-N and probably more portable. WDYT?

Reply:
I think it's actually worth doing. This will actually enable us to support all android versions using the same code. I calculated the math angle on an API 28 Pixel 4 image -- the computation of processTimeSec seemed to be around 2 seconds off the mark. Is that something we can live with?

Reply:
Update: Not valid anymore since we're directly subtracting between process times to get relative values now.